src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const x = Big(numberOne);
+  const y = Big(numberTwo);
+
+  switch (operation) {
+    case '+':
+      return x.plus(y);
+    case '-':
+      return x.minus(y);
+    case 'x':
+      return x.mul(y);
+    case '÷':
+      return x.div(y);
```

Comment:
dont forget to manage division by ZERO on milestone 5, else it will generate an error